### PR TITLE
Added a CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Largely inspired by the same file in the starterkit lessons - thanks folks.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# How to contribute
+
+This [glossary][repo] is an open-source project, and we welcome contributions of all kinds,
+be them new material, corrections or improvements to existing material.
+You are also more than welcome to participate in the reviews of pull requests.
+
+By contributing, you are agreeing that we may redistribute your work under [this license][license].
```
Comment (chrisburr) [inline CONTRIBUTING.md]:
```suggestion
By contributing, you are agreeing that we may redistribute your work under [CC-BY-4.0][license].
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+1.  We use the [fork and pull][gh-fork-pull] model to manage changes. More information
+    about [forking a repository][gh-fork] and [making a Pull Request][gh-pull].
+
+2. You should branch from and submit pull requests against the `master` branch.
+
+3. The glossary is homogeneous in its writing. Please use the following template when writing new entries:
+
+   ```markdown
+
+   ## TERM: The Expanded Real Meaning {#TERM}
```
Comment (chrisburr) [inline CONTRIBUTING.md]: We probably have duplicated terms or acronyms

```suggestion
   ## TERM: The Expanded Real Meaning {#HTML_TAG}
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply CONTRIBUTING.md]: OK.

File: CONTRIBUTING.md (hunk 2 above)
Comment (eduardo-rodrigues) [reply CONTRIBUTING.md]: But `#TERM` is meant to make it clear that it makes sense to have the anchor just as the keyword.

Comment (chrisburr) [reply CONTRIBUTING.md]: Yeah, your probably right that it's clearer LGTM 👍

**Pull request merged**